src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -25,170 +25,185 @@ const propTypes = {
+                onSubmit={submit}
+                style={STYLES.Form}
+                scrollContextEnabled
+            >
+                <Text>{translate('requestorStep.subtitle')}</Text>
+                <View style={STYLES.LearnMoreContainer}>
+                    <TextLink
+                        style={STYLES.LearnMoreLink}
+                        // eslint-disable-next-line max-len
+                        href="https://community.expensify.com/discussion/6983/faq-why-do-i-need-to-provide-personal-documentation-when-setting-up-updating-my-bank-account"
+                    >
+                        {`${translate('requestorStep.learnMore')}`}
+                    </TextLink>
+                    <Text style={STYLES.LearnMoreSeparator}>{' | '}</Text>
+                    <TextLink
+                        style={STYLES.DataSafeLink}
+                        href="https://community.expensify.com/discussion/5677/deep-dive-security-how-expensify-protects-your-information"
+                    >
+                        {`${translate('requestorStep.isMyDataSafe')}`}
+                    </TextLink>
+                </View>
+                <IdentityForm
+                    translate={translate}
+                    defaultValues={defaultValues}
+                    inputKeys={INPUT_KEYS}
+                    shouldSaveDraft
+                />
+                <CheckboxWithLabel
+                    accessibilityLabel={translate('requestorStep.isControllingOfficer')}
+                    inputID="isControllingOfficer"
+                    defaultValue={getDefaultStateForField('isControllingOfficer', false)}
+                    LabelComponent={LabelComponent}
+                    style={STYLES.ControllingOfficerCheckbox}
+                    shouldSaveDraft
+                />
+                <Text style={STYLES.ConditionsContainer}>
+                    {translate('requestorStep.onFidoConditions')}
+                    <TextLink
+                        href="https://onfido.com/facial-scan-policy-and-release/"
+                        style={STYLES.ConditionsLink}
+                    >
+                        {translate('onfidoStep.facialScan')}
+                    </TextLink>
+                    {', '}
+                    <TextLink
+                        href="https://onfido.com/privacy/"
+                        style={STYLES.ConditionsLink}
+                    >
+                        {translate('common.privacy')}
+                    </TextLink>
+                    {` ${translate('common.and')} `}
+                    <TextLink
+                        href="https://onfido.com/terms-of-service/"
+                        style={STYLES.ConditionsLink}
+                    >
+                        {translate('common.termsOfService')}
+                    </TextLink>
+                </Text>
+            </Form>
+        </ScreenWrapper>
+    );
 }
 
+const RequestorStep = React.forwardRef(InnerRequestorStep);
+
```

Comment:
`displayName` is required for functional components.

Reply:
Done ✅